app/setting/controllers/setting-categories-controller.js

```diff
@@ -33,7 +35,10 @@ function (
                 angular.forEach($scope.selectedTags, function (tagId) {
                     calls.push(TagEndpoint.delete({ id: tagId }).$promise);
                 });
-                $q.all(calls).then($scope.refreshView);
+                $q.all(calls).then(function () {
+                    CacheManager.removeCacheGroup('tagCache', '/tags');
```

Comment:
shouldn't this happen in the endpoint itself? 
components outside of an endpoint shouldn't really have to know about internals of its caching.